### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by 2nutz4u** [DeineAgenturUG/greenbone-gvm-openvas-for-docker#38]
End-of-life scan,  current build is 22.4.1. Anyway to update to this version?

**Comment by xenago** [comment]
Right now the easiest move is to use the greenbone community containers, they are updated and 'official':

https://greenbone.github.io/docs/latest/22.4/container/index.html#docker-compose-file

It is not ideal of course, updating in-place would be simplest, but it does work.

**Comment by 2nutz4u** [comment]
Yes, I have seen that and running it parallel as test. But, my only concern with that is no static volume mapping. I tried changing the volumes to static mount points, but it doesn't work. I spent whole day digging through logs trying to figure it out but came up empty.
